src/3.1/entrypoint.sh

```diff
@@ -15,40 +15,39 @@ export LD_LIBRARY_PATH=${MUSIC_PATH}/lib:$LD_LIBRARY_PATH
 export PATH=${MUSIC_PATH}/bin:$PATH
 export CPATH=${MUSIC_PATH}/include:$CPATH
 export PYTHONPATH=${MUSIC_PATH}/lib/python3.8/site-packages:$PYTHONPATH
-export NEST_SERVER_RESTRICTION_OFF=true
-export NEST_SERVER_MODULES=nest,numpy
 
-if [[ ! -d /opt/data ]]; then
-    mkdir /opt/data
-fi
+# Create folder if not existed and then change the directory.
+mkdir_cd () {
+    if [[ ! -d $1 ]]; then
+        mkdir $1
+    fi
+    cd $1
+}
 
-if [[ "$1" = 'notebook' ]]; then
-    cd /opt/data
-    exec jupyter-notebook --ip="${IP_ADDRESS}" --port=8080 --no-browser --allow-root
-fi
+MODE="${mode:-$1}"
+if [[ "${MODE}" = 'interactive' ]]; then
+    read -p "Your python script: " name
+    echo Starting: $name
+    # Start
+    mkdir_cd /opt/data
```

Comment:
Using `mkdir -p` has exactly the effect of the `if` in the function.
```suggestion
    mkdir -p /opt/data
    cd /opt/data
```